content/blog/2022-05-16-may-22-heartbeat.md

```diff
@@ -0,0 +1,393 @@
+posed an interesting point about code quality on LinkedIn. Join the discussion
+and weigh in at this post:
+
+<iframe src="https://www.linkedin.com/embed/feed/update/urn:li:share:6931541880090324992" height="800" width="504" frameborder="0" allowfullscreen="" title="Embedded post"></iframe>
+
+# Company News
+
+## ICYMI: We released TPI! 🎉
+
+On April 27th we released the latest offering to our tool ecosystem.
+
+https://media.giphy.com/media/ut7lqhIfOscbjuU6YQ/giphy.gif
+
+[Terraform Provider Iterative (TPI)](https://tpi.cml.dev) is a Terraform plugin
+built with machine learning in mind. Full lifecycle management of computing
+resources (including GPUs and respawning spot instances) from several cloud
+vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
+
+- **Lower cost with spot recovery:** transparent data checkpoint/restore &
+  auto-respawning of low-cost spot/preemptible instances
+- **No cloud vendor lock-in:** switch between clouds with just one line thanks
+  to unified abstraction
+- **No waste:** auto-cleanup unused resources (terminate compute instances upon
+  task completion/failure & remove storage upon download of results), pay only
+  for what you use
+- **Developer-first experience:** one-command data sync & code execution with no
+  external server, making the cloud feel like a laptop
+
+- ⭐️ [Star the Repo](https://tpi.cml.dev)
+- ✍🏼 [Read the release blog post](https://dvc.org/blog/terraform-provider)
+- ⚙️ Read:
+  [Moving Local Experiments to the Cloud with Terraform Provider Iterative (TPI) tutorial](https://dvc.org/blog/local-experiments-to-cloud-with-tpi)
+- 🎥 [Watch the video](https://www.youtube.com/watch?v=2fEgO8SazSE&t=2s)
+- 🪐
+  [TPI with Jupyter Notebooks Repo](https://github.com/iterative/blog-tpi-jupyter)
+
+Stay tuned for more tutorials and use cases to come!
+
+https://media.giphy.com/media/MrCYIN3x0SgdG/giphy.gif
+
+## 🚀~~Mission Impossible~~ - We have a mission statement!
+
+We did it! This year we surveyed the entire team to arrive at a mission
+statement for Iterative. It was no small feat to decide on what it should be
+given the early stage of our industry, the variety of our tools, and always a
+struggle - figuring out the best and most concise way to convey these ideas (you
+know our penchant for abbreviations). But we persevered and succeeded. Behold
+Iterative's new mission statement:
+
+> We deliver the best developer experience for machine learning teams by
+> creating an ecosystem of open, modular ML tools.
+
+As always the door is open for your feedback on how we can serve your needs
+better!
+
+## ODSC East
+
+We attended our first post-pandemic, in-person conference in Boston last month.
+It was awesome to be together as a team, see
+[**Dmitry Petrov**](https://twitter.com/FullStackML),
+[**Milicia Mc Gregor**](https://twitter.com/FlippedCoding), and
```

Comment:
```suggestion
[**Milecia McGregor**](https://twitter.com/FlippedCoding), and
```

Reply:
Thanks. I keep doing this! 🙈 I did this more than once I think.  I go through the find.